templates/ingress-internal.yaml

```diff
@@ -33,26 +33,29 @@ metadata:
     estafette.io/cloudflare-hostnames: "{{.InternalHostsJoined}}"
 spec:
   tls:
-  - hosts:
-    {{- range .InternalHosts}}
-    - {{.}}
-    {{- end}}
-    {{- if .UseCertificateSecret }}
-    secretName: {{.CertificateSecretName}}
-    {{- else }}
-    secretName: {{.Name}}-letsencrypt-certificate
-    {{- end }}
+    - hosts:
```

Comment:
@snigdhasambitak can you fix extra whitespace? It is better to keep indentation same as per remaining file.

Reply:
resolved